examples/getting_started/run.py

```diff
@@ -12,7 +12,100 @@
 import time
 
 
-NUM_STEPS = 100
+NUM_STEPS = 200
+
+
+# Custom Env implementation for OpenBot
+class OpenBotEnv(spear.Env):
+
+    def __init__(self, config):
+    
+        super().__init__(config)
+        self._wheel_rotation_speeds = None
+    
+    def step(self, action):
+
+        obs, reward, is_done, step_info = super().step(action=action)
+        assert "wheel_encoder" in obs.keys()
+        self._wheel_rotation_speeds = obs["wheel_encoder"]
+
+        return obs, reward, is_done, step_info
+
+    def reset(self, reset_info=None):
+    
+        obs = super().reset(reset_info=reset_info)
+        assert "wheel_encoder" in obs.keys()
+        self._wheel_rotation_speeds = obs["wheel_encoder"]
+
+        return obs
+
+    def _apply_action(self, action):
+
+        # calculate wheel torques based on input PWM signals
+        if "apply_voltage" in action.keys():
+
+            assert self._wheel_rotation_speeds is not None
+            assert len(action["apply_voltage"]) == 2
+
+            duty_cycle = np.array([action["apply_voltage"][0], action["apply_voltage"][1], action["apply_voltage"][0], action["apply_voltage"][1]], dtype=np.float64)
+
+            # Motor torque: 1200 gf.cm (gram force centimeter) == 0.1177 N.m
+            # Gear ratio: 50
+            # Max wheel torque: 5.88399 N.m
+            # https://www.conrad.de/de/p/joy-it-com-motor01-getriebemotor-gelb-schwarz-passend-fuer-einplatinen-computer-arduino-banana-pi-cubieboard-raspbe-1573543.html
+
+            motor_velocity_constant = self._config.OPENBOT_ENV.MOTOR_VELOCITY_CONSTANT # Motor torque constant in [N.m/A]
+            gear_ratio              = self._config.OPENBOT_ENV.GEAR_RATIO              # Gear ratio of the OpenBot motors
+            motor_torque_constant   = 1.0 / motor_velocity_constant                    # Motor torque constant in [rad/s/V]
+
+            battery_voltage         = self._config.OPENBOT_ENV.BATTERY_VOLTAGE         # Voltage of the battery powering the OpenBot [V]
+            control_dead_zone       = self._config.OPENBOT_ENV.CONTROL_DEAD_ZONE       # Absolute duty cycle (in the ACTION_SCALE range) below which a command does not produce any torque on the vehicle
+            motor_torque_max        = self._config.OPENBOT_ENV.MOTOR_TORQUE_MAX        # Motor maximal torque [N.m]
+            electrical_resistance   = self._config.OPENBOT_ENV.ELECTRICAL_RESISTANCE   # Motor winding electrical resistance [Ohms]
+
+            # Speed multiplier defined in the OpenBot to map a [-1,1] action to a suitable command to 
+            # be processed by the low-level microcontroller. For more details, feel free to check the
+            # "speedMultiplier" command in the OpenBot code.
+            # https://github.com/isl-org/OpenBot/blob/master/android/app/src/main/java/org/openbot/vehicle/Vehicle.java#L375
+            action_scale = self._config.OPENBOT_ENV.ACTION_SCALE
```

Comment:
Not used anywhere except to control the dead zone. Is that intended?

Reply:
Ah, this is how things have always been implemented, so we can assume for now that it's correct. Seems weird though.